### PR TITLE
Move flask config env

### PR DESCRIPTION
**Pull request by StepanTchynetskyi**

### CODE REVIEW

**Review by lurak — APPROVED** [review]

**Review by SLDem — APPROVED** [review]

File: tests/stock_dashboard_api/views/test_stock_data_view.py [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+
 from unittest.mock import patch
 
 from stock_dashboard_api import app
 from stock_dashboard_api.models.stock_data_models import StockData
+from stock_dashboard_api.views import stocks_data_view
+
+# Flask app configuration for testing
+app.config.from_object(os.environ.get('FLASK_TESTING_CONFIG'))
+app.register_blueprint(stocks_data_view.test_mod)
```
Comment (alexei-alexov) [inline tests/stock_dashboard_api/views/test_stock_data_view.py]: Why do you need to register blueprint in test module?

File: stock_dashboard_api/views/stocks_data_view.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from stock_dashboard_api.models.stock_data_models import StockData
 from stock_dashboard_api.utils.json_parser import middleware_body_parse_json
 mod = Blueprint('stocks_data', __name__, url_prefix='/stocks_data')
-
+test_mod = Blueprint('test_stocks_data', __name__, url_prefix='/stocks_data')
```
Comment (alexei-alexov) [inline stock_dashboard_api/views/stocks_data_view.py]: Why it's called test_mod?

File: stock_dashboard_api/__init__.py [hunk 3]
```diff
@@ -2,14 +2,12 @@
...
 
-from stock_dashboard_api.config import DevelopmentConfig
 from stock_dashboard_api.views import stock_view, dashboard_views, stocks_data_view, stock_dashboard_view
 
 TEMPLATE_FOLDER = os.path.join(os.environ.get('PROD_ROOT'), 'templates')
 app = Flask(__name__, template_folder=TEMPLATE_FOLDER)
 
-
-app.config.from_object(DevelopmentConfig)
+app.config.from_object(os.environ.get('FLASK_DEVELOPMENT_CONFIG'))
```
Comment (alexei-alexov) [inline stock_dashboard_api/__init__.py]: What if I want to use non DEVELOPMENT config, like stage or prod? Where can I configure which config should I use?

File: tests/stock_dashboard_api/views/test_stock_data_view.py (hunk 1 above)
Comment (StepanTchynetskyi) [reply tests/stock_dashboard_api/views/test_stock_data_view.py]: to test CRUD functionality that prohibited for use by the user

File: stock_dashboard_api/views/stocks_data_view.py (hunk 2 above)
Comment (StepanTchynetskyi) [reply stock_dashboard_api/views/stocks_data_view.py]: it's called test_mod to show that this  Blueprint used only in tests

File: stock_dashboard_api/__init__.py (hunk 3 above)
Comment (StepanTchynetskyi) [reply stock_dashboard_api/__init__.py]: FLASK_DEVELOPMENT_CONFIG changed to FLASK_CONFIG

**Review by daniiloleshchuk — APPROVED** [review]

**Review by lhalam — APPROVED** [review]

**Pull request merged**